Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift

```diff
@@ -181,3 +194,10 @@ extension TuistGraph.Target {
         return (sources: sourcesWithoutPlaygrounds, playgrounds: Array(playgrounds))
     }
 }
+
+extension Array where Element == TuistGraph.ResourceFileElement {
```

Comment:
Nitpick: This is better placed at `ReesourceFileElement.swift` because the utility is scoped to that model. This is key to make this extension more **discoverable**